pkgs/development/python-modules/googlemaps/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub
+, requests
+, responses, pytestCheckHook, pytest, pytestcov
+}:
+
+buildPythonPackage rec {
+  pname = "googlemaps";
+  version = "4.4.2";
+
```

Build fails on python2: 
```
  DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support
in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
  Processing ./googlemaps-4.4.2-py2-none-any.whl
  ERROR: Package 'googlemaps' requires a different Python: 2.7.18 not in '>=3.5'
cannot build derivation '/nix/store/wpfc7hqvj0mcfx6qzfys0l4pn0bjkh7b-env.drv': 1 dependencies couldn't be built
```

```suggestion
  disabled = isPy27;
```